DistributedCollective/Sovryn-frontend
# fix(borrow): change function call to use new abi

Implements: https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/412
Fixes: https://sovryn.monday.com/boards/2254448153/pulses/2261866105

QA:
- borrow some assets in borrow page
- verify you see loan in active debts tab
- verify you can repay it
- verify you can see borrowing action in debts history tab

- if you don't any open positions, then: open margin trading position (use #2017 to open)
- verify you see new position in active positions table
